markdown/blog/helsinki-and-community.md

```diff
@@ -0,0 +1,87 @@
+    byline: AsyncAPI Executive Director
+excerpt: 'Conferences are great. They bring communities together and give people a platform to share their experiences. But is that enough?'
+featured: true
+---
+
+## How It All Started
+
+After three years of hosting online conferences (2020-2022), the AsyncAPI community felt it was time for something different—an in-person event. We wanted to meet people face-to-face, to confirm that behind the virtual interactions were real humans, not bots. Humans we could have meaningful conversations with!
+
+In 2023, we organized multiple events under the brand `AsyncAPI Conf on Tour`. One of them took place at [APIDays Paris](https://www.apidays.global/paris2023/), made possible by the amazing folks behind [APIDays](https://www.apidays.global/), who share our belief in the future of strong API specifications and standards.
+
+It was a fantastic opportunity, but I had one major concern—how do we find our community members in an event with thousands of attendees? How do we have meaningful conversations? And how do we engage with people who don’t yet know about AsyncAPI? As you probably know, conference talks end, and people scatter. It’s easy to lose them in the crowd.
+
+One of the best places to meet people at conferences is at the sponsor booths. But as an open-source, community-driven project, we don’t have the budget for that kind of setup.
+
+Once again, [APIDays](https://www.apidays.global/) came to the rescue and offered us a booth.
+
+We decided to take it one step further—not just a booth for AsyncAPI, but a shared space for other open-source communities. At Paris 2023, we organized the first `API Specs & Standards` booth, bringing together experts from AsyncAPI, OpenAPI, and JSON Schema. We created a vendor-neutral, safe space for knowledge exchange, where experts from different companies shared their expertise without pushing for their products.
+
+## AsyncAPI Conf on Tour at APIDays Helsinki 2024
+
+Our next stop was [APIDays Helsinki 2024](https://www.apidays.global/helsinki_and_north/).
+
+Hugo kicked things off with a great introduction to the AsyncAPI spec and later wrapped up the track with an excellent session on testing event-driven architectures.
+
+<Figure
+  src="/img/posts/helsinki-and-community/hugo.webp"
+  caption="Hugo presenting AsyncAPI basics."
+  className="text-center"
+/>
+
+Jonas followed with a detailed overview of the AsyncAPI tooling ecosystem, showcasing the various tools built around the specification to support developers working with AsyncAPI documents.
+
+<Figure
+  src="/img/posts/helsinki-and-community/jonas.webp"
+  caption="Jonas presenting the AsyncAPI tooling ecosystem."
+  className="text-center"
+/>
+
+I then presented an overview of AsyncAPI use cases, demonstrating how different companies are using AsyncAPI in production, all based on real data we’ve gathered over the years. You can explore all of this in our [AsyncAPI case studies](/casestudies).
```

Comment:
```suggestion
I then presented an overview of AsyncAPI use cases, demonstrating how different companies are using AsyncAPI in production, all based on real data we’ve gathered over the years. You can explore all of this in our [AsyncAPI case studies](https://www.asyncapi.com/casestudies).
```

Reply:
it is going to be posted on asyncapi.com so why not relative link but absolute?

Reply:
Oh I actually didn't know you could use a relative link once merged. Learned something for my future articles